### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by carlosprietobarron**
This is my first pull request on the project Appleclone

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+</head>
+
+<body>
+  <main id="main_viewport">
+
+    <div class="Nav-Bar">
+      <div class="box box1">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
```
Comment (euqueme) [inline index.html]: :no_good_woman:  remove these images in the navbar as the gradient has to be achieved with [CSS Gradients](https://www.w3schools.com/css/css3_gradients.asp)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        </a>
+      </div>
+      <div class="box box2">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
```
Comment (euqueme) [inline index.html]: :no_good_woman:  remove these images in the navbar as the gradient has to be achieved with [CSS Gradients](https://www.w3schools.com/css/css3_gradients.asp)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        </a>
+      </div>
+      <div class="box box3">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
```
Comment (euqueme) [inline index.html]: :no_good_woman:  remove these images in the navbar as the gradient has to be achieved with [CSS Gradients](https://www.w3schools.com/css/css3_gradients.asp)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+        </a>
+      </div>
+      <div class="box box4">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
```
Comment (euqueme) [inline index.html]: :no_good_woman:  remove these images in the navbar as the gradient has to be achieved with [CSS Gradients](https://www.w3schools.com/css/css3_gradients.asp)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+        </a>
+      </div>
+      <div class="box box5">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
```
Comment (euqueme) [inline index.html]: :no_good_woman:  remove these images in the navbar as the gradient has to be achieved with [CSS Gradients](https://www.w3schools.com/css/css3_gradients.asp)

File: index.html [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+        </a>
+      </div>
+      <div class="box box6">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
```
Comment (euqueme) [inline index.html]: :no_good_woman:  remove these images in the navbar as the gradient has to be achieved with [CSS Gradients](https://www.w3schools.com/css/css3_gradients.asp)

File: index.html [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+        </a>
+      </div>
+      <div class="box box7">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
```
Comment (euqueme) [inline index.html]: :no_good_woman:  remove these images in the navbar as the gradient has to be achieved with [CSS Gradients](https://www.w3schools.com/css/css3_gradients.asp)

File: index.html [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+        </a>
+      </div>
+      <div class="box box8">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
```
Comment (euqueme) [inline index.html]: :no_good_woman:  remove these images in the navbar as the gradient has to be achieved with [CSS Gradients](https://www.w3schools.com/css/css3_gradients.asp)

File: index.html [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+      <div class="promo promo2">
+        <img src="./img/promo2.jpg" alt="">
+      </div>
+      <div class="promo promo3">
+        <img src="./img/happy_birthday.jpg" alt="">
+      </div>
+      <div class="promo promo4">
+        <img src="./img/supplier_resp.jpg" alt="">
+      </div>
+    </section>
```
Comment (euqueme) [inline index.html]: :no_good_woman:  remove all **\** elements in the **promo section** as they have to be background images

File: index.html [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+          <a class="menu-right-opt" href="#">Environment</a>
+        </li>
+        <li>
+          <a class="menu-right-opt" href="#">Job Opportunities</a>
+        </li>
+        <li>
+          <a class="menu-right-opt" href="#">Contact Us</a>
+        </li>
+        <li class="worldwide">
+          <a href="#"><img src="./img/flag-btn.png" alt="Choose your country or region" height="16" width="16"></a>
```
Comment (euqueme) [inline index.html]: :memo: please put **heigh="16"** and **width="16"** in your CSSfile

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P.(TSE)

### Hello @carlosprietobarron :wave: 

### Your project looks amazing :art: :eyes:  but is not following the rules of the Odin project and some Microverse specifications, please correct the following:

**Footer**

- [x]  I see you have added the footer but the elements aren't showing

image

#### Promo Section

- [x] The four images of this section should be backgrounds as the project is about building with backgrounds and gradients

#### Navigation background

* [x] It uses CSS [`background -> gradient`](https://www.w3schools.com/cssref/css3_pr_background.asp) for background ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/navigation_requirement.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-backgrounds-and-gradients))

- [x] Correct this CSS bad practice: no inline styling

[OPTIONAL] Change the `<div> `that holds your navbar to be a `<nav> `tag

[OPTIONAL] Follow these [steps](https://help.github.com/en/github/working-with-github-pages/creating-a-github-pages-site) from step #6 to create a Live Demo of your project and add it to your Readme file

#### I have commented your code where lines should be removed :no_good_woman:  or changed :memo: 

### Thats it! :raised_hands: 

### Please :pray: do the changes :memo:  and request back for a code review when you are done :rocket: 

Happy Coding! :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

File: index.html (hunk 1 above)
Comment (carlosprietobarron) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (carlosprietobarron) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (carlosprietobarron) [reply index.html]: done

File: index.html (hunk 4 above)
Comment (carlosprietobarron) [reply index.html]: done

File: index.html (hunk 5 above)
Comment (carlosprietobarron) [reply index.html]: done

File: index.html (hunk 6 above)
Comment (carlosprietobarron) [reply index.html]: done

File: index.html (hunk 9 above)
Comment (carlosprietobarron) [reply index.html]: done

File: index.html (hunk 10 above)
Comment (carlosprietobarron) [reply index.html]: done

**Review by nikasulo — APPROVED** [review]
# This is such a perfect job! @carlosprietobarron :rocket: 
You really pulled this off! Hats off to you :womans_hat: Please merge and keep up this level of work, it will pay off! See you at the very top :medal_military: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by carlosprietobarron** [comment]
Thanks for your review!

Have a great day!

**Pull request merged**